src/modules/transaction/components/TransactionRow/schemas.css

```diff
@@ -25,6 +28,7 @@
       padding: 16px 8px;
       box-sizing: border-box;
       color: var(--color-content-light);
+      font-weight: var(--font-weight-normal);
     }
 
     & > *:nth-child(1) {
```

Comment:
Selector too weak and generic. Same goes for the next few rules.

Reply:
I guess the reason this was done was because the table row can take in any element. to start defining styles based on each element would make the stying too specific making it hard for a third person to refactor if there is a change in component structure